---
bool success   							# indicate successful run of triggered service
uint16[] rooms 							# A list of the rooms 